lib/active_model_serializers/key_transform.rb

```diff
@@ -66,5 +66,28 @@ def underscore(value)
     def unaltered(value)
       value
     end
+
+    # Transforms string to selected case
+    # Accepts string in any case: 'camel', 'undescore', 'dashed'.
+    #
+    # @example:
+    #   transform('SomeClass', :underscore) => 'some_class'
+    #   transform('some_class', :snake) => 'someClass'
+    #   etc...
+    def transform(string, type)
```

Comment:
I recall asking this in an earlier revision but I'll ask again. Why is this `transform` method needed? Why not do it like https://github.com/rails-api/active_model_serializers/blob/d191342a6c2d203ec17fe0ce050f996b50d42596/lib/active_model_serializers/adapter/base.rb#L34 ?